.github/workflows/pkg-version-bump.yaml

```diff
@@ -18,29 +18,36 @@ jobs:
     name: Generate Token
     runs-on: ubuntu-latest
     outputs:
-      token: ${{ steps.generate_token.outputs.token }}
-      app-slug: ${{ steps.generate_token.outputs.app-slug }}
+      token: ${{ steps.generate-bot-token.outputs.token }}
+      app-slug: ${{ steps.generate-bot-token.outputs.app-slug }}
+      user-id: ${{ steps.get-bot-user-id.outputs.user-id }}
     steps:
       - name: Generate release bot app token
-        id: generate_token
+        id: generate-bot-token
         uses: actions/create-github-app-token@v1
         with:
```

Comment:
I think if you're planning on splitting the Token Generation tasks into its own job, you'll need to add `skip-token-revoke` so the post step won't revoke it automatically.  [see here](https://github.com/actions/create-github-app-token?tab=readme-ov-file#skip-token-revoke)

Reply:
> Optional: If truthy, the token will not be revoked when the current job is complete.


So when will it be revoked? When the whole workflow is complet?

Reply:
Alright thanks.
Well, we learned that this was a bad idea all along any way